tests/integration/test_remoted/conftest.py

```diff
@@ -23,3 +26,18 @@ def restart_remoted(get_configuration, request):
         control_service('start', daemon=DAEMON_NAME)
     except sb.CalledProcessError:
         pass
+
+
+@pytest.fixture(scope="module")
+def create_agent_group(group_name='testing_group'):
+    """Temporary creates a new agent group for testing purpose, must be run only on Managers."""
+
+    sb.run([f"{WAZUH_PATH}/bin/agent_groups", "-q", "-a", "-g", group_name])
+
+    with open(f"{WAZUH_PATH}/etc/shared/{group_name}/agent.conf", "w") as agent_conf_file:
+        with open(default_agent_conf_path, 'r') as configuration:
+            agent_conf_file.write(configuration.read())
+
+    yield
+
+    sb.run([f"{WAZUH_PATH}/bin/agent_groups", "-q", "-r", "-g", group_name])
```

Comment:
I think is better to move this to the `wazuh_testing` module as a new function and use that function inside the fixture.

Reply:
https://github.com/wazuh/wazuh-qa/pull/1115/commits/fe573623294f35376566b4ff0d29c08a9482753b